src/tests/gui/Nav.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { render, screen, fireEvent } from "@testing-library/react";
```

Comment:
For `tests` folder we should follow the same folder structure and filenames as we do inside `src`. This shall help other developers to locate unit tests for the components in `src`.

Reply:
Okay sure, would do